settings.gradle.kts

```diff
@@ -254,6 +254,7 @@ include(":instrumentation:hibernate:hibernate-6.0:javaagent")
 include(":instrumentation:hibernate:hibernate-6.0:spring-testing")
 include(":instrumentation:hibernate:hibernate-common:javaagent")
 include(":instrumentation:hibernate:hibernate-procedure-call-4.3:javaagent")
+include(":instrumentation:hibernate:hibernate-reactive:javaagent")
```

Comment:
Shouldn't the module be named `hibernate-reactive-1.0`? Muzzle matches all versions, right, but there's nothing earlier than 1.0 on maven central anyway

Reply:
renamed